components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -298,14 +298,14 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
                     <div className="flex flex-col mt-4 p-4 rounded-xl bg-gray-50 dark:bg-gray-800">
                         <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Current Plan</div>
                         <div className="mt-1 text-xl font-semibold flex-grow text-gray-600 dark:text-gray-400">
-                            Free
+                            {usageLimit > 500 ? "Professional Open Source" : "Free"}
```

Comment:
**thought**: Cross-posting [from the relevant RFC](https://www.notion.so/RFC-Professional-OSS-Plan-for-UBP-d7af057f0b6042b3af92c12e7cb00179?d=0efd576190b9432091ba118efe97b263#c52b6cf88e4a4f8490934356cc0088a4) (internal) in case simplifying this could be ok:

> @gtsiolis: Maybe worth considering renaming this to simply _**Open Source**_? Possible reasons:
> 1. There’s now no seat-based plan named open source, see [#13229](https://github.com/gitpod-io/gitpod/issues/13229).
> 2. Making the plan name smaller, easier to pronounce, identify, etc.
> 3. Focus on the open source aspect.

Also, :a: using the term professional could imply some sort of employment status which can be non-applicable and :b: open source matches with the program name[[1](https://www.gitpod.io/for/opensource)].